### ISSUE TITLE
.wr after .wa removes different amount

### ISSUE THREAD

**Issue by diogo-mustachio-barros** [diogo-mustachio-barros/better-wishlist#1]
Example:
```
.wa Fate/Grand Order || Nitocris (Alter), Tlaloc, Kirschtaria Wodi-, Napoleon, White Holy Grail, Goddess Rhongomy-, Ashtart Origin, Ibaraki-douji, Edward Teach, BB Pele, Nebuchadnezzar II, Nikola Tesla, Paul Bunyan, Pino, Priestess of the-, Scandinavia Pepe-, Shibata Katsuie, Thomas Edison, Yagyuu Munenori, William Tell
```

Adds 20 cards, but changing to `.wr` only removes 15.

**Comment by diogo-mustachio-barros** [comment]
Left out cards appear to be: 
```
Goddess Rhongomy-
Kirschtaria Wodi-
Nebuchadnezzar II
Priestess of the-
Scandinavia Pepe-
```

Perhaps it is because of the `-` character of the hard 16 character limit for search terms

**Comment by diogo-mustachio-barros** [comment]
Names where not correctly being trimmed to size 16 on insertion. On search we would try to match 16 characters against 17 and it wouldn't work

Fixed on commit a46be38
